Product/Production/Gateway/AdminGUIWebservices/src/main/java/gov/hhs/fha/nhinc/configadmin/ConfigAdmin.java

```diff
@@ -721,4 +739,49 @@ public SimpleCertificateResponseMessageType importToTruststore(ImportToTruststor
         return buildSimpleResponse(true, "Import is successful into the temporary truststore.");
```

Comment:
"Import is successful into the temporary truststore." -> "Import into the temporary truststore is successful."